### PR TITLE
Update elements to the latest version of the protocol spec

### LINKED ISSUE #262: Update backend to support new Element types
See http://dev.sanamobile.org/users/procedures/#date

New types:
- `DATE`
- `PLUGIN`
- `ENTRY_PLUGIN`

Removed types:
- `GPS`
- `SOUND`

### PR DESCRIPTION
**Pull request by Kapin**
Shouldn't break the front-end...

Closes #262

### CODE REVIEW

File: src-django/api/models.py [hunk 1]
```diff
@@ -64,13 +64,13 @@ class Meta:
...
+        ('ENTRY', 'ENTRY'),
         ('SELECT', 'SELECT'),
         ('MULTI_SELECT', 'MULTI_SELECT'),
         ('RADIO', 'RADIO'),
-        ('GPS', 'GPS'),
-        ('SOUND', 'SOUND'),
         ('PICTURE', 'PICTURE'),
-        ('ENTRY', 'ENTRY')
+        ('PLUGIN', 'PLUGIN'),
+        ('ENTRY_PLUGIN', 'ENTRY_PLUGIN')
```
Comment (Trinovantes) [inline src-django/api/models.py]: You haven't implemented the `DATE` type yet

Comment (Kapin) [inline src-django/api/models.py]: F

**Comment by Ommy** [comment]
+1

**Comment by Trinovantes** [comment]
+1

**Pull request merged**